charts/fleet-crd/templates/crds.yaml

```diff
@@ -2091,6 +2091,8 @@ spec:
               pollingInterval:
                 nullable: true
                 type: string
+              pruneBundles:
+                type: boolean
```

Comment:
as discussed offline, we should move this into an option in the `fleet apply` command and by default we should be pruning. If we want, we can provide this as an option like `--keep-bundles`

Reply:
I do not think we can just add the option to `fleet apply` cli without backing up with the API change - all the existing options of the apply command are API fields.
I think we can add prune by default for now.